src/modules/menu/components/toolboxRight/GeolocButton.vue

```diff
@@ -1,13 +1,15 @@
 <template>
-    <div
+    <button
```

Comment:
:+1: for changing to `button` (dunno if there was a reason not to use button, but `button` seems right)